_layouts/tag-archive.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: base
+---
+
+<h1>Archive of posts with {{ page.type }} '{{ page.title }}'</h1>
```

Comment:
```suggestion
<h1>Posts with {{ page.type }} '{{ page.title }}'</h1>
```

I don't like Archive as a name.